core/app/help.go

```diff
@@ -23,6 +23,45 @@ import (
 	"github.com/google/gapid/core/app/flags"
 )
 
+// MakeDocs
+func MakeDoc(ctx context.Context, name string){
+	filename := name + ".md"
+	w, _ := os.Create(filename)
+	globalVerbs.selected = nil
+	fmt.Fprintln(w, "**Note**: Autogenerated documentation, do not modify!")
+	fmt.Fprintln(w, "# GAPIT Help")
+
+	// Print list of verbs
+	fmt.Fprintln(w, "| Command | Short help |")
+	fmt.Fprintln(w, "| -- | -- |")
+	format := fmt.Sprintf("|[%%s](#%%s)|%%s|")
+	backlink := "[Back](#gapit-help)"
+	for _, child := range globalVerbs.verbs {
+		fmt.Fprintf(w, format, child.Name, child.Name, child.ShortHelp)
+		fmt.Fprintln(w)
+	}
+
+	for _, verb := range globalVerbs.verbs {
+		globalVerbs.selected = verb
+		// Header
+		fmt.Fprintln(w, "# ", verb.Name)
+		fmt.Fprintln(w, backlink)
+		fmt.Fprintln(w)
+		// Usage
+		fmt.Fprintf(w, "*%s*", verb.ShortHelp)
+		fmt.Fprintln(w)
+		fmt.Fprintln(w)
+		fmt.Fprint(w, "**Usage:** ")
+		verbUsage(w, &globalVerbs, true)
+		// Full flags
+		fmt.Fprintln(w, "```")
+		verbHelp(w, &globalVerbs, true)
+		fmt.Fprintln(w, "```")
+		fmt.Fprintln(w, backlink)
```

Comment:
So you have a backlink after the title and at the end of the full flag section, do you also want the one at the end? (I'm OK either way, just to point it out)

Reply:
Fixed.